.github/workflows/VercelDeploy.yaml

```
@@ -37,19 +39,34 @@ jobs:
+      - name: Check content of folder
+        run: ls -al
+
       - name: Pull Vercel Environment
         run: |
-          npx vercel pull --yes --environment=preview --token=${{ secrets.VERCEL_CLI_TOKEN }}
+          npx vercel pull --yes --environment=preview --token=$VERCEL_CLI_TOKEN
           echo "Vercel Environment pulled successfully."
 
       - name: Build project
-        run: vercel build --yes --token=${{ secrets.VERCEL_CLI_TOKEN }}
+        run: vercel build --yes --token=$VERCEL_CLI_TOKEN
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:28: Double quote to prevent globbing and word splitting

(shellcheck)